todo-app/src/ToDo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import TodoDesc from './TodoDesc.js'
+import ToDoStatus from './ToDoStatus.js'
+import './todo.css'
+
+
+export default class TODO extends React.Component {
```

Comment:
It would be nice for the class name to reflect the file name, for example `class ToDo`